docs/releases/slashing-release/interfaces/core/slashing-parameter-configuration.md

```diff
@@ -0,0 +1,222 @@
+     * @param allocationDelay the delay in seconds for the operator's allocations
+     */
+    struct AllocationDelayDetails {
+        bool isSet;
+        uint32 allocationDelay;
+    }
+
+    /// EVENTS
+
+    event MagnitudeAllocated(
+        address operator,
+        IStrategy strategy,
+        OperatorSet operatorSet,
+        uint64 magnitudeToAllocate,
+        uint32 effectTimestamp,
+    );
+
+    event MagnitudeDeallocated(
+        address operator,
+        IStrategy strategy,
+        OperatorSet operatorSet,
+        uint64 magnitudeToDeallocate,
+        uint32 completeableTimestamp,
+    );
+
+
+    /// EXTERNAL - STATE MODIFYING
+    
+    /**
+     * @notice Modifies the proportions of slashable stake allocated to a list of operatorSets for a set of strategies
+     * @param operator address to modify allocations for
+     * @param allocations array of magnitude adjustments for multiple strategies and corresponding operator sets
+     * @param operatorSignature signature of the operator if msg.sender is not the operator
+     * @dev updates freeMagnitude for the updated strategies
+     * @dev must be called by the operator
+     */
+    function modifyAllocations(
+        address operator,
+        MagnitudeAllocation[] calldata allocations,
+        SignatureWithSaltAndExpiry calldata operatorSignature
+    ) external;
+
+    /**
+     * @notice For all pending deallocations that have become completable, their pending free magnitude can be
+     * added back to the free magnitude of the (operator, strategy) amount. This function takes a list of strategies
+     * and adds all completable deallocations for each strategy, updating the freeMagnitudes of the operator
+     *
+     * @param operator address to complete deallocations for
+     * @param strategies a list of strategies to complete deallocations for
+     *
+     * @dev can be called permissionlessly by anyone
+     */
+    function updateFreeMagnitude(
+        address operator,
+        IStrategy[] calldata strategies,
+        uint8[] calldata numToComplete
+    ) external;
+
+    /**
+     * @notice Called by operators to set their allocation delay. Can only be set one time.
+     * @param delay the allocation delay in seconds
+     * @dev this is expected to be updatable in a future release
+     */
+    function initializeAllocationDelay(uint32 delay) external;
+
+    /// VIEW
+
+    /**
+     * @param operator the operator to get the slashable bips for
+     * @param operatorSet the operatorSet to get the slashable bips for
+     * @param strategy the strategy to get the slashable bips for
+     * @param timestamp the timestamp to get the slashable bips for for
+     *
+     * @return slashableBips the slashable bips of the given strategy owned by
+     * the given OperatorSet for the given operator and timestamp
+     */
+    function getSlashableBips(
```

Comment:
imo we should just return parts per 1e18 that the AVS is allowed to slash. rounding with bips just too sketch

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/709/commits/df5dd0a437851e2dcd792e17b6c5146bee48fec4